react-native/README.md

```diff
@@ -21,13 +21,16 @@ We are iterating quickly and a lot of the code is changing every day.
 
 #### General (both android and ios)
 
+Follow the [React Native Getting Started guide](https://facebook.github.io/react-native/docs/getting-started.html) for the platforms you intend to develop on.
+
 ```sh
 # Setup golang mobile
 go get golang.org/x/mobile/cmd/gomobile
 gomobile init
 
 # Setup npm
 npm install
+npm-install -g react-native-cli
```

Comment:
@chrisnojima this was necessary for the `react-native run-android` command to work. I tried running `~/node_modules/.bin/react-native` after my initial npm install, but it was just a placeholder that instructed me to install `react-native-cli`.